### PR TITLE
Add an upgrade guide for jQuery 3.5.0

### PR DESCRIPTION
**Pull request by mgol**

### CODE REVIEW

File: pages/download.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ To locally download these files, right-click the link and select "Save as..." fr
 
 ### jQuery
 
-For help when upgrading jQuery, please see the [upgrade guide](https://jquery.com/upgrade-guide/) most relevant to your version.
```
Comment (mgol) [inline pages/download.md]: This makes it easier to test locally with Vagrant.

**Comment by mgol** [comment]
Migrate PR: https://github.com/jquery/jquery-migrate/pull/328.

File: pages/download.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ To locally download these files, right-click the link and select "Save as..." fr
...
+For help when upgrading jQuery, please see the [upgrade guide](/upgrade-guide/) most relevant to your version.
```
Comment (mgol) [inline pages/download.md]: This makes it easier to test locally with Vagrant.

File: pages/upgrade-guide/3.5.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	"title": "jQuery Core 3.5 Upgrade Guide"
+}</script>
+
+## jQuery.htmlPrefilter changes
+
+### A workaround
+
+If you want to upgrade to jQuery 3.5.0 or newer and don't have time to deal with breaking changes at the moment and you use jQuery Migrate 3.2.0 or newer, you can revert to the previous behavior by invoking:
+```js
+jQuery.restoreLegacyInsecureHtmlPrefilter();
```
Comment (timmywil) [inline pages/upgrade-guide/3.5.md]: Update here when ready.

Comment (mgol) [reply pages/upgrade-guide/3.5.md]: Thanks for the reminder. Done.

**Pull request merged**